### ISSUE TITLE
[Legitimate Site Blocked] markmeta.finance

### ISSUE THREAD

**Issue by MarkMeta** [MetaMask/eth-phishing-detect#6521]
https://markmeta.finance/

**Comment by MarkMeta** [comment]
MarkMeta is an upcoming metaverse project which is in the development state right now. Kindly review it and unblock the site.

**Comment by devenblake** [comment]
Duplicate of #6516. Please don't file duplicate issues.

**Comment by MarkMeta** [comment]
Kindly review the project's website and unblock the site.
https://markmeta.finance/

Note: The project is still in the development state.
